lib/ansible/module_utils/six/__init__.py

```diff
@@ -33,6 +33,14 @@
 import sys
 import types
 
+from ansible.module_utils.common import warnings as _warnings
+
+_warnings.deprecate(
+    msg="The `ansible.module_utils.six` module is deprecated.",
+    help_text="Use the Python standard library equivalent instead.",
+    version="2.24",
```

Comment:
This is very agressive and not helpful, since now this warning will show up to many, many users, and collections will not be able to fix this while they're still supporting older ansible-core versions like 2.16.

Showing a deprecation when running ansible-test would be a lot more helpful than this.